vignettes/ebola.Rmd

```diff
@@ -0,0 +1,620 @@
+  mutate(
+    sex = case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  facet_grid(district ~ sex) +
+  theme_minimal()
+```
+
+### Continuous probability density function
+
+The posterior predictive distribution under no truncation and no censoring.
+That is to produce continuous delay times.
+I don't like that `geom_density()` is doing kernel smoothing but plotting as a histogram is suggestive of there being discrete structure.
+
+No stratification:
+
+```{r}
+newdata_pdf <- obs_prep |>
+  as.data.frame() |>
+  mutate(obs_t = 1000, pwindow_upr = 0.001, swindow_upr = 0.001)
+
+draws_pdf <- predicted_draws(fit, newdata = newdata_pdf, ndraws = 1000)
+```
+
+```{r}
+ggplot(draws_pdf, aes(x = .prediction)) +
+  geom_density() +
+  labs(x = "PDF with no censoring and no truncation", y = "") +
+  theme_minimal()
+```
+
+Stratification by sex:
+
+```{r}
+newdata_sex_pdf <- obs_prep |>
+  as.data.frame() |>
+  data_grid(sex) |>
+  mutate(obs_t = 1000, pwindow_upr = 0.001, swindow_upr = 0.001)
+
+draws_sex_pdf <- predicted_draws(
+  fit_sex,
+  newdata = newdata_sex_pdf,
+  ndraws = 1000
+)
+```
+
+```{r}
+draws_sex_pmf |>
+  mutate(
+    sex = case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_density() +
+  labs(x = "PDF with no censoring and no truncation", y = "") +
+  facet_grid(. ~ sex) +
+  theme_minimal()
+```
+
+Stratification by sex:
+
+```{r}
+newdata_sex_district_pdf <- obs_prep |>
+  as.data.frame() |>
+  data_grid(sex, district) |>
+  mutate(obs_t = 1000, pwindow_upr = 0.001, swindow_upr = 0.001)
+
+draws_sex_district_pdf <- predicted_draws(
+  fit_sex_district,
+  newdata = newdata_sex_district_pdf,
+  ndraws = 1000
+)
+
+```{r}
+draws_sex_district_pdf |>
+  mutate(
+    sex = case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_density() +
+  labs(x = "PDF with no censoring and no truncation", y = "") +
+  facet_grid(district ~ sex) +
+  theme_minimal()
+```
+
+## Model comparison
+
+* Use `loo` and PSIS
+* Find out if it's worth including sex as a covariate. Presumably yes given how different the delay distributions are
+* Find out if it's worth including district as a random effects
+  * What about spatially structured? Probably future work: how to include more structured random effects
+
+# Using a gamma delay distribution
+
+## Fitting 
+
+* Just fit the best model from above here?
+* Going to do the sex stratified one for now
+
+```{r}
+fit_sex_gamma <- epidist(
+  data = obs_prep,
+  formula = brms::bf(mu ~ 1 + sex, shape ~ 1 + sex),
+  family = stats::Gamma(),
+  algorithm = "laplace",
+  refresh = 0,
+  silent = 2,
+  seed = 1
+)
+
+summary(fit_sex_gamma)
+```
+
+# Model selection
```

Comment:
I think these concepts are covered above, so could skip this or make it really short

Reply:
I've cut this section into https://github.com/epinowcast/epidist/issues/285